sdk/src/env.rs

```diff
@@ -254,6 +254,89 @@ impl Env {
     pub fn invoke_contract(&mut self, hf: xdr::HostFunction, args: xdr::ScVec) -> xdr::ScVal {
         self.env_impl.invoke_function(hf, args).unwrap()
     }
+
+    #[cfg(not(target_family = "wasm"))]
+    fn clone_self_and_catch_panic<F, T>(&self, f: F) -> (Env, std::thread::Result<T>)
+    where
+        F: FnOnce(Env) -> T,
+    {
+        let hook = std::panic::take_hook();
+        std::panic::set_hook(Box::new(|_| ()));
```

Comment:
❔ What's the significant of setting a panic handling that does nothing during the clone? If a panic occurs during clone for some reason won't it be gobbled and ignored?